lib/license_finder/package_managers/yarn.rb

```diff
@@ -66,7 +70,12 @@ def prepare_command
     private
 
     def yarn2_prepare_command
```

Comment:
The prepare_command on line 62 only checks for yarn1 or 2. What is it supposed to be if it is yarn 3? I also don't like how this yarn2 prepare command is for yarn 3 as well. This file needs to be refactored

Reply:
the check for yarn2 actually is checking for any versions greater or equal to yarn2, so it includes yarn 3. I was originally trying to change as little things as possible, but I agree it is confusing. I'll do some refactoring

Reply:
made the refactor